packages/coordinator/src/screens/bookedDonation/BookedAppointmentScreen.tsx

```diff
@@ -33,7 +41,30 @@ export default function BookedAppointmentScreen(
         stickyComponent: <div>{props.appointment.firstName}</div>,
       }}
     >
-      {JSON.stringify(props)}
+      <div className={styles.details}>פרטי התורם</div>
+      <InfoBar
+        title={"מספר טלפון"}
+        value={
+          <a href={`tel: ${props.appointment.phone}`}>
+            {props.appointment.phone}
+          </a>
+        }
+        icon={<Phone />}
+      />
+      <InfoBar
+        title={"סוג דם"}
+        value={getValueInDiv(props.appointment.bloodType)}
```

Comment:
Use `LocaleUtils.getBloodTypeTranslation()` to get the blood type string.

See how it is imported in other files